src/jump.js

```diff
@@ -92,6 +106,21 @@ const jumper = () => {
     easing   = options.easing   || easeInOutQuad
     a11y     = options.a11y     || false
 
+    // resolve container
+    switch(typeof options.container) {
+      case 'object':
+        // we assume container is an HTML element (Node)
+        container = options.container
+        break
+
+      case 'string':
+        container = document.querySelector(options.container)
```

Comment:
Wouldn’t it be better to look up the container going from the `target` up instead of from the `document` down?

```
container = target.closest(options.container)
```

It would exclude errors where the container selector would be a common class name instead of id. Also might be faster.